src/main/scala/ai/privado/passes/PropertyEnvLinkerPassBase.scala

```diff
@@ -0,0 +1,34 @@
+package ai.privado.passes
+
+import ai.privado.languageEngine.java.language.NodeStarters
+import ai.privado.tagger.PrivadoParallelCpgPass
+import io.shiftleft.codepropertygraph.generated.{Cpg, EdgeTypes}
+import io.shiftleft.codepropertygraph.generated.nodes.JavaProperty
+import io.shiftleft.codepropertygraph.generated.nodes.*
+import org.slf4j.LoggerFactory
+import overflowdb.{BatchedUpdate, Node}
+
+/** This pass creates a graph layer for env call from property node
+  */
+
+abstract class PropertyEnvLinkerPassBase(cpg: Cpg) extends PrivadoParallelCpgPass[JavaProperty](cpg) {
+
+  val logger = LoggerFactory.getLogger(getClass)
+  override def generateParts(): Array[_ <: AnyRef] = {
+    cpg.property.iterator
```

Comment:
As discussed, this is incorrect, as in `generateParts` the call where we intend to link Ex - `getEnv` calls `annotations` needs to be added, because these are limited in number as compared to propertyNodes, which can easily shoot up

Reply:
yes right, I was testing something, i will make the change and once the PR is ready for review, i will notify you.

Reply:
Yes done.